source/benchmark_data.cc

```diff
@@ -34,10 +34,10 @@ DFGBechmarkRequest<dim>::DFGBechmarkRequest
  const double reference_length)
 :
 Re(reynolds_number),
-front_evaluation_point(0.15 / reference_length,
-                       0.20 / reference_length),
-rear_evaluation_point(0.25 / reference_length,
-                      0.20 / reference_length),
+front_evaluation_point(1.5 / reference_length,
+                       2.0 / reference_length),
+rear_evaluation_point(2.5 / reference_length,
+                      2.0 / reference_length),
```

Comment:
This is wrong. More specifically, the default value of `reference_length` is wrong. It should be `0.1`. The other values remain unchanged.

Reply:
I don't understand your comment. The cylinder has a diameter of 1 which means that we scale the spatial coordinates specified by the DFG benchmark by a factor of 10. Therefore the original coordinates of the evaluation points (0.15, 0.2) and (0.25, 0.2) become (1.5, 2.0) and (2.5, 2.0). I removed the variable `reference_length` in da2d96e.